charts/victoria-metrics-operator/templates/cluster_role.yaml

```diff
@@ -0,0 +1,269 @@
+{{- if .Values.rbac.create -}}
```

Comment:
kindly ask you to add psp option:

https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-agent/templates/podsecuritypolicy.yaml

Reply:
will add in separate PR